study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/study/questionnaire.jsp

```diff
@@ -474,10 +477,11 @@
               Date/Time of launch (pick one)
               <span class="requiredStar">*</span>
               <span
-                  class="ml-xs sprites_v3 filled-tooltip"
+              <fmt:formatDate value = "${date}" pattern="z" var="server_timezone"/>
+                  class="ml-xs sprites_v3 filled-tooltip Selectedtooltip"
                   data-toggle="tooltip"
                   data-placement="bottom"
-                  title="All date/time selections are based on server timezone.">
+                  title="Selected dates and times will work as per the mobile device time. Selections of dates or times in the past (as per the server timezone which is ${server_timezone}) is not allowed.">
```

Comment:
get help from tech writer for all text like this (see comments in other file)

Reply:
Fixed